src/net_processing.cpp

```diff
@@ -4275,7 +4275,7 @@ bool PeerManagerImpl::ProcessMessages(CNode* pfrom, std::atomic<bool>& interrupt
         pfrom->ConnectionTypeAsString().c_str(),
         msg.m_type.c_str(),
         msg.m_recv.size(),
-        msg.m_recv.data()
+        (unsigned char*)(msg.m_recv.data())
```

Comment:
Note to self: I want to check if this causes a difference in the number of instructions for this tracepoint. I think a [`reinterpret_cast`](https://en.cppreference.com/w/cpp/language/reinterpret_cast) could also work here as an alternative:

> Unlike static_cast, but like const_cast, the reinterpret_cast expression does not compile to any CPU instructions [..]